Fix handling of Interface SVGTextPositioningElement

msand opened the pull request.

msand committed:
Fix handling of Interface SVGTextPositioningElement

msand committed:
Merge remote-tracking branch 'upstream/master' into AddTSpan

# Conflicts:
#	index.js

3rd-Eden commented on package.json:
```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```
Why did it need stage-2 here? I don't see anything here using it, or am I missing something obvious here.

msand replied:
It uses spread properties for object literals. It is stage-3 now, so could actually change that.

3rd-Eden replied:
Shouldn't that just be the https://babeljs.io/docs/plugins/transform-object-rest-spread/ plugin then instead of a stage 2/3 if that is the only thing it needs.

msand replied:
True, replaced stage-3 with babel-plugin-transform-object-rest-spread.

3rd-Eden commented:
Code LGTM, just one question about a dependency.

msand committed:
Fix linting. Change babel-preset-stage-2 to stage-3.

Remove deprecated godaddy-style, replace with eslint-config-godaddy-react, fix errors.

3rd-Eden commented on index.js:
```diff
@@ -27,29 +28,29 @@ function copypaste(from, to, ...props) {
-  //
-  // Correctly apply the transformation properties.
-  //
+    //
```
I don't think these should be indented.

3rd-Eden commented on index.js:
```diff
@@ -27,29 +28,29 @@ function copypaste(from, to, ...props) {
-  // attribute. So we can safely conditionally add things to our `style` object
-  // and re-introduce it to our `clean` object
-  //
+    //
```
Looks like this indent is off as well.

msand replied:
Hmm, not sure how that happened, didn't touch any indentation manually, must've been some formatting tool applied by mistake. Reverted.

msand committed:
Fix formatting.

msand committed:
Fix formatting.

msand committed:
Replace babel-preset-stage-3 with babel-plugin-transform-object-rest-spread.

3rd-Eden commented:
Thanks for your contribution, it will be in the next release <3
